src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,60 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String model;
+    private int year;
+    private Engine engine;
+    private String color;
+    private int maxSpeed;
 
+    public Car(String model, int year, Engine engine, String color, int maxSpeed) {
+        this.model = model;
+        this.year = year;
+        this.engine = engine;
+        this.color = color;
+        this.maxSpeed = maxSpeed;
+    }
+
+    @Override
+    public Car clone() {
+        try {
+            Car clonedCar = (Car) super.clone();
+            clonedCar.setEngine(engine.clone());
+            return clonedCar;
+        } catch (CloneNotSupportedException e) {
+            throw new RuntimeException("This car can't be cloned", e);
+        }
+    }
+
+    public String toString() {
+        StringBuilder stringBuilder = new StringBuilder("Model: ");
```

Comment:
let's use the default String concatenation

Reply:
done